doc/whats_new/v1.1.rst

```diff
@@ -0,0 +1,52 @@
+.. include:: _contributors.rst
+
+.. currentmodule:: sklearn
+
+.. _changes_1_1:
+
+Version 1.1.0
+=============
+
+**In Development**
+
+
+.. include:: changelog_legend.inc
+
+Minimal dependencies
+--------------------
+
+Version 1.1.0 of scikit-learn requires python 3.7+, numpy 1.14.6+ and
+scipy 1.1.0+. Optional minimal dependency is matplotlib 2.2.2+.
```

Comment:
do we only state matplotlib or pandas and co. to be able to run the examples?

Reply:
Technically [all those](https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/_min_dependencies.py#L22) are optional dependencies, so it's unclear where we put the limit. I'm fine with the current version and also fine with adding pandas.

Reply:
I also thought about that, but as far as it goes for the library, and not the examples, we only consume pandas, whereas we actually use matplotlib. So I figured I'm happy with having matplotlib here but not pandas. Once we give pandas as output, then we'd need to have it here.